src/app/modules/convai-checker/perspective-status.component.ts

```diff
@@ -46,13 +46,11 @@ export enum Emoji {
 
 export enum Configuration {
   DEMO_SITE,
-  EXTERNAL,
 };
 
 // The keys in ConfigurationInput should match items in the Configuration enum.
 export const ConfigurationInput = {
```

Comment:
Can we just remove this now? Seems that the abstraction here is no longer doing anything for us?

Reply:
I want to keep this here so we have the ability to add other configurations later. We originally added this in a pretty big refactor because there were lots of rapid iterations on the design of the checker and we wanted to be able to accommodate more than one. Keeping this allows for adding more later if we want to.

Reply:
ok! Maybe add a comment for that then, and I'm happy.